.circleci/config.yml

```diff
@@ -559,6 +562,9 @@ commands:
                   echo "Installing rustup"
                   & $installer_dir\rustup-init.exe --profile minimal -y
                   exit $LASTEXITCODE
+            - run:
+                name: Configure rustup
+                command: rustup show active-toolchain; if ($LASTEXITCODE -ne 0) { rustup toolchain install }
```

Comment:
NIT: why is this one different than the other ones?

Reply:
Essentially because we're using Windows PowerShell here and they don't possess the Unix `||` operator. As this StackOverflow post explains: https://stackoverflow.com/questions/2416662/what-are-the-powershell-equivalents-of-bashs-and-operators